graphql/data/marathon/groups/server.js

```diff
@@ -0,0 +1,40 @@
+import DataLoader from 'dataloader';
+
+import config from '../../../config';
+import { fetchWithAuth } from '../../../utils/fetch';
+
+export default class GroupsServerConnector {
+  constructor({ authToken }) {
+    const query = {
+      embed: [
+        'group.groups',
+        'group.apps',
+        'group.pods',
+        'group.apps.deployments',
+        'group.apps.counts',
+        'group.apps.tasks',
+        'group.apps.taskStats',
+        'group.apps.lastTaskFailure'
+      ]
+    };
+
+    const getGroup = fetchWithAuth(authToken, {
+      baseURI: config.endpoints.marathon.groups,
+      query
+    });
+
+    this.groupsLoader = new DataLoader(getGroup, { batch: false });
+  }
+
+  get(groupId = '/') {
```

Comment:
Yes the reason I put the default argument in is so it can be called with no arguments. Otherwise we explicitly have to query with `groupId: '/'` which might not be such a bad thing anyway. I can remove the default.